src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -99,7 +99,8 @@
                     Spacing="24">
 
             <!--  Select Color and Add New Button  -->
-            <StackPanel Orientation="Horizontal">
+            <StackPanel Margin="0,5,0,0"

```

Comment:
this also moves the whole box down -- OK?

Reply:
Yes, very slightly! The page doesn't look any different from the other pages though (and it is going to be overhauled very soon anyway)